src/credentials/CHIPCert.cpp

```diff
@@ -765,6 +772,32 @@ CHIP_ERROR ChipDN::GetCertType(uint8_t & certType) const
     return err;
 }
 
+CHIP_ERROR ChipDN::GetCertChipVal(uint64_t & chipVal) const
+{
+    uint8_t rdnCount = RDNCount();
+
+    chipVal = 0;
+
+    for (uint8_t i = 0; i < rdnCount; i++)
+    {
+        switch (rdn[i].mAttrOID)
```

Comment:
perhaps `IsChipX509Attr()` can be used instead?

Reply:
We can't use that because that includes Fabric ID, and we only want to retrieve the ChipVal associated with either Root/ICA/Node/FirmwareID. Basically everything except Fabric ID.